doc/release/1.13.0-notes.rst

```diff
@@ -344,6 +344,13 @@ New ``positive`` ufunc
 This ufunc corresponds to unary `+`, but unlike `+` on an ndarray it will raise
 an error if array values do not support numeric operations.
 
+New ``divmod`` ufunc
+--------------------
+This ufunc corresponds to the Python builtin `divmod`, and is used to implement
+`divmod` when called on numpy arrays. ``np.divmod(x, y)`` calculates a result
+equivalent to ``(np.floor_divide(x, y), np.remainder(x, y))`` but is
+approximately twice as fast as calling the functions separately.
```

Comment:
And before release I'm going to gather all the new ufuncs into a `New ufuncs` section. There are enough of them in the 1.13 release to justify that.